.env.development

```diff
@@ -17,7 +17,7 @@ MARKETING_SITE_BASE_URL='http://localhost:18000'
 ORDER_HISTORY_URL='http://localhost:1996/orders'
 REFRESH_ACCESS_TOKEN_ENDPOINT='http://localhost:18000/login_refresh'
 SEGMENT_KEY=''
-SITE_NAME='Your Platform Name Here'
+SITE_NAME=''
```

Comment:
@attiyaIshaque Is this removed intentionally?

Reply:
@shafqatfarhan Remove by mistake.